src/extensions/export.ts

```diff
@@ -8,6 +8,22 @@ import { humanReadable } from "../deploy/extensions/deploymentSummary";
 import { logger } from "../logger";
 import { FirebaseError } from "../error";
 
+/**
+ * stripProjectId searchs spec.params for any param that include projectId,
+ * and replaces it with a parameterized version that can be used on other projects.
+ * For example, 'my-project-id.appspot.com' becomes '${param:PROJECT_ID}.appspot.com`
+ */
+export function stripProjectId(projectId: string, spec: InstanceSpec): InstanceSpec {
+  const newParams: Record<string, string> = {};
+  for (const [key, val] of Object.entries(spec.params)) {
+    const parameterizedVal = val.replace(projectId, "${param:PROJECT_ID}");
```

Comment:
Seems a bit hacky, what if my project ID is `appspot` or something like it? Do you have an example of a list of params so we can see how common this problem can be?

Reply:
sgtm, just making sure it won't break too often :)